index.html

```diff
@@ -4610,9 +4634,23 @@ <h3>User Agent Processing</h3>
 						href="https://www.w3.org/TR/html/grouping-content.html#the-ul-element"><code>ul</code></a>
 					element.</p>
 
-				<p>The following algorithm MUST be applied to a walk of a DOM subtree rooted at the first
-						<code>nav</code> element in document order with the <code>role</code> attribute value
-						<code>doc-toc</code>. All explanations are <em>informative</em>.</p>
+				<p>The following algorithm MUST be applied to a walk of a DOM subtree rooted at the table of contents
+					element determined as follows:</p>
+
+				<ol>
+					<li>
+						<p>if the <code>LinkedResource</code> that identifies the table of contents includes a fragment
+							identifier in its URL, the element referenced by the fragment identifier provided the
+							element has the <code>role</code> attribute value <code>doc-toc</code>.</p>
```

Comment:
I believe it should say:

> element referenced by the fragment identifier, provided the element has the...

(comma was missing between 'identifier' and 'provided'